crosshair/__init__.py

```diff
@@ -1,7 +1,16 @@
+"""Statically analyze Python code using symbolic execution."""
```

Comment:
I had to change the description to conform with PEP 257.

Reply:
FYI, I've been shying away from the "static analysis" and "dynamic analysis" text in various places. The problem with CrossHair is that people familiar with either often want to say that CrossHair is the other thing.
I do intend it to be used like you'd use a static analysis tool, though, so feel free to assert this. :)